.gitignore

```diff
@@ -1,3 +1,134 @@
+# Custom
+
 /.keys/*
 !/.keys/*.enc
 !/.keys/*.pub
+
+
+# Created by https://www.gitignore.io/api/linux,osx,smalltalk,sublimetext,vim,windows
+
+### Linux ###
+*~
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+
+### OSX ###
+*.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
```

Comment:
Does this work? Are the \r's included in this commit? Will they be preserved upon checkout?